client/src/components/locations.jsx

```diff
@@ -1,33 +1,17 @@
 import React from 'react';
-import axios from 'axios';
 import Location from './location.jsx';
 import { SliderContainer, SliderCardContainer } from './styledComponents.js';
 
 class Locations extends React.Component {
```

Comment:
since you aren't using state anymore you could use a functional component instead of a class component...don't know if there are any tradeoffs except general boilerplate coding for the constructor etc.